Software_Carbon_Intensity/Software_Carbon_Intensity_Specification.md

```diff
@@ -112,46 +112,35 @@ The entity calculating software carbon intensity MUST report what is included wi
 
 ## Methodology Summary 
 
-This standard can be used to calculate the real-world emissions associated with software by measuring the total change in global emissions associated with a particular piece of software.
+The SCI is a rate, carbon emissions per one unit of `R`. The equation used to calculate the Software Carbon Intensity (SCI) of a software system is:
 
-Electricity has a carbon intensity depending on where and when it is consumed. An intensity is a rate. It has a numerator and a denominator. A rate provides you with helpful information when considering the growth of a software product and allows for the computation of a marginal rate.
+`SCI = ((E * I) + M) per R`
 
-To calculate the carbon intensity the following information is needed:
-
-`O`= Operational emissions of a given piece of software
+Where:
 
-`E`= Energy consumed by a given piece of software
+- `E` = Energy consumed by a software system
 
-`I`= Location-based marginal carbon emissions
+- `I`= Location-based marginal carbon emissions
 
-`O = E * I ` = Operational emissions based on energy consumption (E) and location-based carbon intensity measurement (I)
+- `M` = Embodied emissions of a software system
 
-`M` = Embodied emissions of a given piece of software
-****
-**These are used to calculate total carbon emissions (`C`) and carbon intensity (`CI`):**
+- `R` = Baseline as a denominator (e.g. carbon per additional user, API-call, ML job, etc) 
```

Comment:
```suggestion
- `R` = Baseline as a denominator (e.g. carbon per additional user, API-call, ML job, etc) 
```
```suggestion
- `R` = Baseline (e.g. carbon per additional user, API-call, ML job, etc) 
```

Reply:
@jawache since we've removed the mention of numerator and denominator, might be good to remove that word from here as well now.